internal/service/kustomize/alertmanager/README.md

```diff
@@ -0,0 +1,10 @@
+# Service for Alertmanager
```

Comment:
can we move this under `/alarms/k8s` to assert that these only relevant for  alarms service? (and eventually replace them with actual Go code as needed when integrating with operator)

Reply:
done